src/formatDate.js

```diff
@@ -50,7 +50,46 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const days = 'DD';
+  const months = 'MM';
+  const years = 'YYYY';
+  const shortYears = 'YY';
+
+  const daysFromIndex = fromFormat.indexOf(days);
+  const monthsFromIndex = fromFormat.indexOf(months);
+  let yearsFromIndex = fromFormat.indexOf(years);
+
+  const daysToIndex = toFormat.indexOf(days);
+  const monthsToIndex = toFormat.indexOf(months);
+  let yearsToIndex = toFormat.indexOf(years);
```

Comment:
It can be simplify. Divider index is always 3. Month index is always 1. So you need only to find daysIndex, and yearsIndex will be last of 4 elements automatically

Reply:
Hi, thanks for check) But tests failed when month index is not one. And also i don't know how to find last element index(of year) can you help me with this?
Attach screen with failed test below.)
<img width="667" alt="Знімок екрана 2023-01-12 о 13 09 30" src="https://user-images.githubusercontent.com/113355096/212052161-9fa470f2-8c48-4d76-ad20-ce727b8a3a5b.png">